### ISSUE TITLE
Content - Screenreader component

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#540]

**Comment by ethanWallace** [comment]
[Draft document](https://docs.google.com/document/d/12TjNT0pr516pUcxfxo4Jk1kR4VczqQxJyQ_p86fn5aI/edit?usp=sharing)

**Comment by SmartMouthWords** [comment]
Did first pass of content. Added some suggestions and questions.

**Comment by SmartMouthWords** [comment]
Did initial review of [screenreader documentation](https://docs.google.com/document/d/12TjNT0pr516pUcxfxo4Jk1kR4VczqQxJyQ_p86fn5aI/edit)

Follow up with @ethanWallace booked for 7-Feb-2023

**Comment by SmartMouthWords** [comment]
- Bringing the name to content critique this afternoon to help decide if it should be written: screen reader only, screenreader only, screen reader-only, or screenreader-only – Right now it's the only English component with more than 2 words in the name (besides theme and topic menu: we had no control over the name)

- Rewrote the first guidance section and added some things
- Updated the definition 
There's a section in the code and accessibility area on how the component can help read the right language I asked @ethanWallace to look at
- The preview image and anatomy (if we decide to have one) could benefit from some visual design attention

**Comment by SmartMouthWords** [comment]
@EliseKa Ready for translation

**Comment by EliseKa** [comment]
Here's the [translation](https://docs.google.com/document/d/1rr1fPDLQpmM9qzbRMr_loWTTwfg0UIvYqu4I4aWBluw/edit#heading=h.30j0zll) for the screenreader-only component.

**Comment by melaniebmn** [comment]
@adorayi @SmartMouthWords I started working on adding the guidance for the screenreader-only component. We are currently still missing these design/content/translation pieces before the guidance can be published:

- Preview image + alt text
- French translation for component preview: “Text only seen by assistive technologies”
- Matching Figma component that we can link to
- Anatomy
- Anatomy image + alt text
- The current translation doc uses “Structure” as the heading for the anatomy section. We normally add the name of the component to the heading as well which would be “Screenreader-only anatomy” in this case. We need “Screenreader-only anatomy” translated.

**Comment by melaniebmn** [comment]
Here is the [PR](https://github.com/cds-snc/gcds-docs/pull/304) for adding the guidance

**Comment by melaniebmn** [comment]
Idea for what the preview image could look like:

<img width="964" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMXI0QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7937f99fd93c04e3ed2710d466dbccb97bd2d48a/Screenshot%202024-02-28%20at%204.12.03%E2%80%AFPM.png" alt="Screenshot 2024-02-28 at 4.12.03 PM.png" />

**Comment by adorayi** [comment]
@SmartMouthWords  to review the list, what's missing (alt text for anatomy, preview image text)

**Comment by melaniebmn** [comment]
@SmartMouthWords updated the list - let me know if I missed something and we already got some translations back :)
